### DISCUSSION TITLE
AVAST blocks GDBUS.exe in Audacious ver 4.4.1 for Windows, but not in 4.3.1

### DISCUSSION THREAD

**Discussion by Ghastleigh** [audacious-media-player/audacious#1479] [General]
So I just downloaded portable ver 4.4.1 for windows and when starting the program, AVAST blocked and sent GDBUS.exe to quarantine.

What's that all about?

**Comment by jlindgren90** [comment]
gdbus.exe is legitimate. It is an IPC service used e.g. to open files in the running instance of Audacious. I suggest contacting Avast and ask why it was flagged.

**Comment by radioactiveman** [comment]
gdbus is not part of Audacious itself (source code in our responsibility), but the library GLib which Audacious depends on.
I would also suggest to contact Avast for this false positive, we can't resolve that on our side. A quick search led to this [forum thread](https://forum.avast.com/index.php?topic=14433.msg1289438#msg1289438).

Please let us know when they give an update or answer.

**Comment by Fr-Dae** [comment]
> So I just downloaded portable ver 4.4.1 for windows and when starting the program, AVAST blocked and sent GDBUS.exe to quarantine.
> 
> What's that all about?

Disable avast, quickfix ;) it's almost useless
